### ISSUE TITLE
Reduce downloaded data from firebase to reduce cost

### ISSUE THREAD

**Issue by laurentS** [mapswipe/mapswipe#212]
About 2/3 of the costs of firebase realtime database are linked to download of data from the backend. This is billed by volume, so number of requests doesn't really matter, if we're able to lower the number of bytes received.

A few ideas from the app side:
- most of the volume is concentrated in the `tasks`: ideally, we could remove the tasks entirely from the DB, as the url template for the imagery is already in the project level data
- to test this out, we could simply leave the tasks in the DB. We then interpolate each task from looping from one corner of the area to the opposite corner, and generate each url from there.
- nothing needs to be changed in the backend to try this out, so the upgrade path is super easy: just upgrade the app, and once everyone is on the new version, remove tasks from backend.

**Comment by Hagellach37** [comment]
The good thing is, that this change would help to reduce (a) the bytes transfered **and** (b) the bytes stored.

When generating  the url for the tiles you have to consider the following:
* *normal* tile server use x, y, z coordinates for the tiles (this can be our default)
* *google* style tile servers are like normal ones but use an inverted y coordinates (e.g. maxar uses this format)
* *quadkey* tile servers use a quadkey instead of x, y, z coordinates (e.g. Bing)
* *wmts* tile servers might need an additional input on the name of the layer to be displayed

Basically this has been implemented in the back end here:
https://github.com/mapswipe/python-mapswipe-workers/blob/7307b53e9b86d872588313558d94427d13e10dc0/mapswipe_workers/mapswipe_workers/utils/tile_functions.py#L75

**Comment by laurentS** [comment]
Fixed in 2.0.4
